e2e-tests/cypress/e2e/SendCoins.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Send coins
+  As a user
+  I want to send and receive GDD
+  I want to see transaction details on overview and transactions pages
+
+  # Background:
+  #   Given the following "users" are in the database:
+  #     | email                  | password | name               |
+  #     | bob@baumeister.de      | Aa12345_ | Bob Baumeister     |
+  #     | raeuber@hotzenplotz.de | Aa12345_ | Räuber Hotzenplotz |
+  
+  Scenario: Send GDD to other user
```

Comment:
can we do two transactions and check both on receiver side?

Bob -> Hotzenplotz
Bob -> Peter 
Hotzenplotz -> Peter

Expect Bob to have 2 outgoing Transactions
Expect Hotzenplotz to have 1 incoming, 1 outgoing Transaction
Expect Peter to have 2 incomming Transactions

Also I'd like to see the account balance to be tested - this would allow us to ensure decay is calculated correctly (e.g balance < 120.00 GDD && balance >= 119.97 GDD).

I am fine to postpone this, but I believe a more complex test could increase coverage of usecases and catch more errors: I suggest - if you agree with the sketched szenario - to create an issue with that and merge this as you currently implemented it.

Reply:
Yes, that might be a broader coverage of the send coin feature for end-to-end testing.
Please create an issue with your suggestion.

Reply:
https://github.com/gradido/gradido/issues/3076